package.json

```diff
@@ -34,7 +34,7 @@
     "test:exports:docker": "docker compose -f environment_tests/docker-compose.yml up --force-recreate",
     "example": "yarn workspace examples start",
     "precommit": "turbo run precommit",
-    "docs": "yarn workspace core_docs start",
+    "docs": "cd docs/core_docs && yarn workspace core_docs start",
```

Comment:
don't believe you need this `cd` because you already have `yarn workspace`

Reply:
Oh, yeah prob not